src/app/design/frontend/base/default/template/payment/form/omise/omiseoffsiteinternetbankingpayment.phtml

```diff
@@ -56,8 +57,11 @@ $code = $this->getMethodCode();
             </div>
             <div class="omise-banking-text-wrapper">
                 <span class="title">Bangkok Bank</span><br/>
-                <span class="rate secondary-text">Fee: 10 THB (same zone), 20 THB (out zone)</span>
+                <!-- <span class="rate secondary-text">Fee: 10 THB (same zone), 20 THB (out zone)</span> -->
```

Comment:
why we need that commented?
it is what github is for..to remember that stuff ;)
I think we could simply delete that commented lines

Reply:
I forgot to delete 😝